shell/pages/auth/login.vue

```diff
@@ -147,8 +146,6 @@ export default {
       // For newer versions this will return all settings if you are somehow logged in,
       // and just the public ones if you aren't.
       try {
-        await fetchInitialSettings(this.$store);
```

Comment:
Do we want to remove these instances of `fetchInitialSettings()`? I'm considering if they exist to act as a "refresh" in the event that something might have changed behind the scenes.

Reply:
For `fetchInitialSettings` to act as a refresh instead of using our cache the dispatch methods used from within would need to be passed `opt:  { force: true }`.  

Neither dispatch method seems to use this option so I figured that these usages aren't actually refreshing anything. If we'd like to put them back in for comfort I'm fine to do it but the above is why I decided to remove it.

Reply:
Your explanation helps to quell my fears and I'm not seeing a scenario where removing these instances causes a regression while testing this change. I think that we can leave them out.

Reply:
@codyrancher @rak-phillip 

Settings are a special case. Requests when unauthed will result in a partial response, and when authed the full set.

To resolve this the UI uses `load: _MULTI` resulting in the response being cached in the store... but not mark as having all results. So any subsequent calls to findAll will still make a http request (without specifying `force`).

`fetchInitialSettings` was put in place because we were fetching settings multiple times in different scenarios with the wrong props (resulting in multiple blocking requests on every fresh load).

Given the move of fetching settings outside of the authenticated middlware in https://github.com/rancher/dashboard/pull/11165 it looks ok to remove these additional ones from brand, setup and login. However we need to validate
- setup flow works fine
- no additional settings requests are made (if unauthed 1 then another on log in, or only a single one on load when authed)
- parts that depends on settings continue to work (for example the brand is correctly applied on setup and logging pages)

Reply:
@richard-cox thanks for the info. A few questions about the additional testing:

> setup flow works fine

- Are the tests in `dashboard/cypress/e2e/tests/setup/rancher-setup.spec.ts` sufficient for doing this?

> parts that depends on settings continue to work (for example the brand is correctly applied on setup and logging pages)

- Do you mean something else other than setting up rancher (like in the tests I referenced above) when you say setup? 
- What branding will be available before you've setup rancher? Which logging pages are you referring to? 
  - I have already verified that colors and favicons still work on the login page pre auth.

Quick note:
I've just verified that this is working as expected:

> no additional settings requests are made (if unauthed 1 then another on log in, or only a single one on load when authed)

Reply:
> Are the tests in dashboard/cypress/e2e/tests/setup/rancher-setup.spec.ts sufficient for doing this?

Good point! I wish whoever put them in remembered he put them in :man_facepalming: Apologies for churn.

>Do you mean something else other than setting up rancher (like in the tests I referenced above) when you say setup?

Maybe we can skip this one. Branding (/v3/settings/ui-brand) will be applied either at install so will be seen in setup, or once setup in the UI (and setup is not returned to).

> What branding will be available before you've setup rancher? Which logging pages are you referring to?
> - I have already verified that colors and favicons still work on the login page pre auth.

That's the one

> no additional settings requests are made (if unauthed 1 then another on log in, or only a single one on load when authed)

Perfect.

I think all bases are covered then, and it's really nice removing all those extra calls.